stdlib/http/src/main/ballerina/src/http/redirect/redirect_client.bal

```diff
@@ -50,7 +50,7 @@ public type RedirectClient client object {
     # + path - Resource path
     # + message - An optional HTTP outbound request message or any payload of type `string`, `xml`, `json`,
     #             `byte[]`, `io:ReadableByteChannel` or `mime:Entity[]`
-    # + return - The HTTP `Response` message, or an error if the invocation fails
+    # + return - The `http:Response` message, or an `http:ClientError` if the invocation fails
```

Comment:
```suggestion
    # + return - The `http:Response` message, or an else `http:ClientError` if the invocation fails
```

Reply:
Should be "or else an" and remove the comma.